packages/compass-app-stores/package.json

```diff
@@ -57,8 +57,8 @@
     "@mongodb-js/mocha-config-compass": "^1.3.9",
     "@mongodb-js/prettier-config-compass": "^1.0.2",
     "@mongodb-js/tsconfig-compass": "^1.0.4",
-    "@testing-library/dom": "8.20.1",
-    "@testing-library/react": "12.1.5",
+    "@testing-library/dom": "^8.20.1",
+    "@testing-library/react": "^12.1.5",
```

Comment:
Drive-by, noticed these dependencies were exact matches while we usually use the `Compatible with version caret` https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies